addons/missionTesting/XEH_postInit.sqf

```diff
@@ -30,6 +30,7 @@ GVAR(MissionTestingChecklistMaster) = [
         [
             ["Ensure vehicle gear is correct for the side using it (Mainly for COOPs), or removed (mainly for TvTs)",D_CHECK,MISSION_TYPE_APPLIES_BOTH]
             ,["Ensure the vehicles are balanced (I.E. if one team has a BTR, one team has the AT to kill it, or if two teams are racing to an objectives, their vics go about the same speed)",D_CHECK,MISSION_TYPE_APPLIES_BOTH]
+            ,["Ensure names of vehicle roles match Mission Makers' intent for qualifications to apply (ie. - if there's a 'Co-Pilot' role, it requires a Pilot Qual to slot)",D_CHECK,MISSION_TYPE_APPLIES_BOTH]
```

Comment:
Maybe just "asset roles" instead of vehicle roles? Could cover things like MMG being labeled as HMG or etc 
That might require further elaboration/explanation though, so take it or leave it

Reply:
If changed to "asset roles", I think we'd want to move it from under the "vehicles checklist" section to like the "other considerations checklist" section since it's more generic

imo, I don't think we need to specify "asset roles" though as MMG/HMG and even IFV Gunner/Commander delineation is clear or soft enough and historically hasn't been a problem; the original intention behind this line was to solidify the policy update where "Co-Pilots" needed flight qual and "Gunner" and "Crew Chief" roles did not